src/compiler/factory/utilities.ts

```diff
@@ -162,7 +162,7 @@ namespace ts {
             const left = createExpressionFromEntityName(factory, node.left);
             // TODO(rbuckton): Does this need to be parented?
             const right = setParent(setTextRange(factory.cloneNode(node.right), node.right), node.right.parent);
-            return setTextRange(factory.createPropertyAccessExpression(left, right), node);
+            return setOriginalNode(setTextRange(factory.createPropertyAccessExpression(left, right), node), node);

```

Comment:
`transformers/ts.ts` replaces import equals entity names with synthetic variable declarations + property access: https://github.com/microsoft/TypeScript/blob/766e83f92b44ab5c75f309ccd144434d441ceb89/src/compiler/transformers/ts.ts#L2999

I experimented with resolving the synthetic property access, but settled on saving and resolving the original entity name instead?